src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -272,6 +272,16 @@ to operate at the non-zero type;
 extensions will specify how software expresses this authority
 for types not defined above.
 
+Capability encodings may also make the set of <<sec_cap_type>> values
+that may be used to seal a particular capability
+depend on the the permissions granted by that capability.
+For example, it can be a useful space optimization to differentiate
+the <<sec_cap_type>> values
+with which capabilities granting <<x_perm>> may be sealed from those
+with which capabilities not granting <<x_perm>> may be sealed.
+(The <<app_cap_description>> does not avail itself of this option,
+but, for example, the CHERIoT capability encoding does.)
```

Comment:
I am finding it a bit difficult to parse this sentence.

I believe the intent 1) is that the architectural capability type can be different depending on combination of granted permissions and the raw capability type field and 2) when you seal a capability the output can depend on the permissions of the input: E.g. YSENTRY on a cap with x permission gives a "code sentry" and without x permission you get a "sealed data cap"?

I guess we need a "Fixes: https://github.com/riscv/riscv-cheri/issues/64" in the commit message?